tensorflow/python/training/monitored_session_test.py

```diff
@@ -1414,6 +1414,12 @@ def test_merge_caller_and_hook_run_options(self):
             isinstance(hook.run_metadata_list[0], config_pb2.RunMetadata))
         self.assertGreater(len(hook.run_metadata_list[0].partition_graphs), 0)
 
+  def test_with_statement_and_close(self):
+    # Test case for 12224
```

Comment:
nit: end comments with periods per google python style. 
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

Also, the number "12224" doesn't mean much. Include the full github issue link instead.